ci-operator/config/openshift/managed-cluster-config/openshift-managed-cluster-config-master.yaml

```diff
@@ -17,6 +17,59 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+tests:
+- as: checklinks-pr
```

Comment:
Is there any way we can change this PR check to only check the diff'd lines?

We're seeing an issue in the `ops-sop` repo where links are broken that were not changed, which will lead to backlogs of updates not being pushed because some link in another file is broken. It will then also promote having seemingly random changes being made to multiple files in the same PRs just to get them to be able to merge.

IMO, these blocking PR checks should only block if you're attempting to push a new link that is broken. Otherwise the weekly check should be sufficient.